fcrepo-configs/src/main/resources/config/file-s3/repository.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name" : "repo",
+    "jndiName" : "",
+    "workspaces" : {
+        "predefined" : ["default"],
+        "default" : "default",
+        "allowCreation" : true,
+        "cacheSize" : 10000
+    },
+    "storage" : {
+        "persistence": {
+            "type": "file",
+            "path" : "${fcrepo.object.directory:target/objects}"
+        },
+        "binaryStorage" : {
+            "type" : "s3",
+            "username" : "${aws.accessKeyId}",
```

Comment:
The main alternative that I'm familiar with is storing this kind of sensitive information in a file (which has more robust protection from other users).  I don't know if there is any support for reading Java system properties from a file, or if we'd need to manage that process ourselves.